src/main/java/com/breskul/bring/dump/service/PrinterService.java

```diff
@@ -0,0 +1,5 @@
+package com.breskul.bring.dump.service;
```

Comment:
As I understand the `com.breskul.bring.dump` package uses only for tests. I think it should be in the `test` directory.

Reply:
agree